datasets/german_legal_entity_recognition/german_legal_entity_recognition.py

```diff
@@ -0,0 +1,159 @@
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+"""
+_HOMEPAGE_URL = "https://github.com/elenanereiss/Legal-Entity-Recognition"
+_CITATION = """\
+@inproceedings{leitner2019fine,
+  author = {Elena Leitner and Georg Rehm and Julian Moreno-Schneider},
+  title = {{Fine-grained Named Entity Recognition in Legal Documents}},
+  booktitle = {Semantic Systems. The Power of AI and Knowledge
+                  Graphs. Proceedings of the 15th International Conference
+                  (SEMANTiCS 2019)},
+  year = 2019,
+  editor = {Maribel Acosta and Philippe Cudré-Mauroux and Maria
+                  Maleshkova and Tassilo Pellegrini and Harald Sack and York
+                  Sure-Vetter},
+  keywords = {aip},
+  publisher = {Springer},
+  series = {Lecture Notes in Computer Science},
+  number = {11702},
+  address = {Karlsruhe, Germany},
+  month = 9,
+  note = {10/11 September 2019},
+  pages = {272--287},
+  pdf = {https://link.springer.com/content/pdf/10.1007%2F978-3-030-33220-4_20.pdf}}
+"""
+_DATA_URL = "https://raw.githubusercontent.com/elenanereiss/Legal-Entity-Recognition/master/data/dataset_courts.zip"
+_VERSION = "1.0.0"
+_COURTS = ["bag", "bfh", "bgh", "bpatg", "bsg", "bverfg", "bverwg"]
+_COURTS_FILEPATHS = {court: f"{court}.conll" for court in _COURTS}
+_ALL = "all"
+
+
+class GermanLegalEntityRecognitionConfig(datasets.BuilderConfig):
+    def __init__(self, *args, courts=None, **kwargs):
+        super().__init__(*args, version=datasets.Version(_VERSION, ""), **kwargs)
+        self.courts = courts
+
+    @property
+    def filepaths(self):
+        return [_COURTS_FILEPATHS[court] for court in self.courts]
+
+
+class GermanLegalEntityRecognition(datasets.GeneratorBasedBuilder):
+    BUILDER_CONFIGS = [
+        GermanLegalEntityRecognitionConfig(name=court, courts=[court], description=f"Court. {court}.")
+        for court in _COURTS
+    ] + [GermanLegalEntityRecognitionConfig(name=_ALL, courts=_COURTS, description=f"All courts included.")]
+
```

Comment:
```suggestion
    BUILDER_CONFIG_CLASS = GermanLegalEntityRecognitionConfig
```
we need that to specify that we're using a custom config